src/loaders/trackedEntityLoader.js

```diff
@@ -170,7 +170,13 @@ const trackedEntityLoader = async config => {
             }
         );
 
-        data = toGeoJson(dataWithRels.primary);
+        // Show all TEIs expect those that are shown as secondary instances
+        data = toGeoJson(
+            instances.filter(
+                tei => !dataWithRels.secondary.find(s => s.id === tei.id)
+            )
+        );
+
```

Comment:
I think this should work, but I'm a little worried about performance - this has O(n^2) complexity which isn't great.  Should we remove the `primary` array calculation or update it to include instances without relationships?  In any case, we could probably improve performance later if it's important to get this in.